exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -223,19 +224,94 @@ public void connect(Properties props) throws RpcException {
     connect(null, props);
   }
 
+  /**
+   * Function to Populate the endpointList with the list of drillbits
+   * provided in the connection string by client.
+   *
+   * For direct connection we can get URL string having drillbit property as below:
```

Comment:
Nit: this is a Javadoc comment, so strings must be formatted using HTML. One handy tool:

    <dl>
    <dt>drillbit=...</dt>
    <dd>us the ip and port...</dd>
    </dl>

In Eclipse, hovering over the method name shows the formatted Javadoc for a quick check.